addons/dragon/anim/model.cfg

```diff
@@ -0,0 +1,124 @@
+            "camera","Pelvis",// case has changed for arma3
+            "weapon","Spine1",
+            "launcher","Spine1",    
+            //Head skeleton in hierarchy
+            "Neck","Spine3",
+            "Neck1","Neck",
+            "Head","Neck1",
+
+        //Left upper side
+            "LeftShoulder","Spine3",
+            "LeftArm","LeftShoulder",
+            "LeftArmRoll","LeftArm",
+            "LeftForeArm","LeftArmRoll",
+            "LeftForeArmRoll","LeftForeArm",
+            "LeftHand","LeftForeArmRoll",
+            "LeftHandRing","LeftHand",
+            "LeftHandRing1","LeftHandRing",
+            "LeftHandRing2","LeftHandRing1",
+            "LeftHandRing3","LeftHandRing2",
+            "LeftHandPinky1","LeftHandRing",
+            "LeftHandPinky2","LeftHandPinky1",
+            "LeftHandPinky3","LeftHandPinky2",
+            "LeftHandMiddle1","LeftHand",
+            "LeftHandMiddle2","LeftHandMiddle1",
+            "LeftHandMiddle3","LeftHandMiddle2",
+            "LeftHandIndex1","LeftHand",
+            "LeftHandIndex2","LeftHandIndex1",
+            "LeftHandIndex3","LeftHandIndex2",
+            "LeftHandThumb1","LeftHand",
+            "LeftHandThumb2","LeftHandThumb1",
+            "LeftHandThumb3","LeftHandThumb2",
+        //Right upper side
+            "RightShoulder","Spine3",
+            "RightArm","RightShoulder",
+            "RightArmRoll","RightArm",
+            "RightForeArm","RightArmRoll",
+            "RightForeArmRoll","RightForeArm",
+            "RightHand","RightForeArmRoll",
+            "RightHandRing","RightHand",
+            "RightHandRing1","RightHandRing",
+            "RightHandRing2","RightHandRing1",
+            "RightHandRing3","RightHandRing2",
+            "RightHandPinky1","RightHandRing",
+            "RightHandPinky2","RightHandPinky1",
+            "RightHandPinky3","RightHandPinky2",
+            "RightHandMiddle1","RightHand",
+            "RightHandMiddle2","RightHandMiddle1",
+            "RightHandMiddle3","RightHandMiddle2",
+            "RightHandIndex1","RightHand",
+            "RightHandIndex2","RightHandIndex1",
+            "RightHandIndex3","RightHandIndex2",
+            "RightHandThumb1","RightHand",
+            "RightHandThumb2","RightHandThumb1",
+            "RightHandThumb3","RightHandThumb2",
+        //Left lower side
+            "LeftUpLeg","Pelvis",
+            "LeftUpLegRoll","LeftUpLeg",
+            "LeftLeg","LeftUpLegRoll",
+            "LeftLegRoll","LeftLeg",
+            "LeftFoot","LeftLegRoll",
+            "LeftToeBase","LeftFoot",
+        //Right lower side
+            "RightUpLeg","Pelvis",
+            "RightUpLegRoll","RightUpLeg",
+            "RightLeg","RightUpLegRoll",
+            "RightLegRoll","RightLeg",
+            "RightFoot","RightLegRoll",
+            "RightToeBase","RightFoot",
+
+            //New facial features arma3 only
+            "Face_Hub","Head",
+                "Face_Jawbone","Face_Hub",
```

Comment:
Different indentation

Reply:
That's intended and correct. This is a tree-like structure of the bones of the skeleton.
```
base
 base->left_leg
  left_leg->left_foot
 base->right_leg
  right_lef->right_foot
```
@TheCandianVendingMachine revert pls.

Reply:
I don't know what the indentation should be, I'm just copying from `gestures/anim/model.cfg`

Reply:
I made a pull request with correct indentation according to the skeleton. I linked it above.
But if it's just a copy you might aswell just leave it, otherwise the original that was copied should also be fixed.

Reply:
I agree that it's a nice formatting fix and I have fixed all other instances in #6796